config/translate.go

```diff
@@ -301,12 +303,14 @@ func TranslateFromV2_0(old v2_0.Config) types.Config {
 	}
 
 	for _, oldFile := range old.Storage.Files {
+		uid := oldFile.User.Id
+		gid := oldFile.Group.Id
 		file := types.File{
 			Node: types.Node{
 				Filesystem: oldFile.Filesystem,
 				Path:       string(oldFile.Path),
-				User:       types.NodeUser{ID: oldFile.User.Id},
-				Group:      types.NodeGroup{ID: oldFile.Group.Id},
+				User:       types.NodeUser{ID: &uid},
```

Comment:
`intToPtr` was written with constant values in mind, but I guess it would work here. I originally wanted to use `&oldFile.User.Id`, but that doesn't work due to go's `range` reusing the variable address.